phpcs.xml.dist

```diff
@@ -112,6 +112,13 @@
 
 	<rule ref="Yoast.NamingConventions.NamespaceName">
 		<properties>
+			<!-- Treat the "src" directory as the project root for path to namespace translations. -->
```

Comment:
Very minor: I'd prefer `path-to-namespace translations` over `path to namespace translations` because it makes it easier to mentally parse the sentence.

Reply:
Also: this comment only covers one of the values below. What about 'config' and 'tests/unit'?

Reply:
I've updated the comment to cover this better.